package.json

```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "@babel/plugin-proposal-nullish-coalescing-operator": "*",
     "@babel/plugin-proposal-optional-chaining": "*",
+    "@commitlint/cli": "^12.1.4",
```

Comment:
Maybe you should put this version in master-theme too ? (it's 12.1.1 in the other PR)

Reply:
Oh right. They release a new version since I first tried this out :grin: